src/compositions/Admin/LoginRegister.css

```diff
@@ -1,31 +1,36 @@
-.LoginBox, .RegisterBox {
+.Login,
```

Comment:
Should LoginRegister.css be in the src/compositions/Login folder instead of the Admin folder?

Reply:
yes good catch and this should be a request changes :)